contracts/datafeeds/DataFeedInterface.sol

```diff
@@ -10,6 +10,8 @@ contract DataFeedInterface {
 
     // Get registartion specific information
     function isDataSet(address ofAsset) constant returns (bool) {}
+    function isDataValid(address ofAsset) constant returns (bool) {}
+    function isValid(address ofAsset) constant returns (bool) {}
```

Comment:
@travs `isDataValid` and `isValid` is the same function - different only in different pre_conditions!
that's why above function was internal in the first place.
so essentially all the changes in DataFeed and DataFeedInterface should be reverted